lib/localDev.ts

```diff
@@ -323,14 +323,14 @@ export async function createDeveloperTestAccountForLocalDev(
       accountId
     );
 
-    const result = await buildDeveloperTestAccount(
+    const devTestAccountId = await buildDeveloperTestAccount(
       name,
       accountConfig,
       env,
       maxTestPortals
     );
 
```

Comment:
Nit, we can just directly return the result and not use the intermediate variable.